components/src/core/InfoListItem/InfoListItem.tsx

```diff
@@ -339,13 +339,9 @@ export const InfoListItem: React.FC<InfoListItemProps> = (props) => {
         if (avatar) {
             return statusColor
                 ? color(statusColor).isDark()
-                    ? // @ts-ignore TODO
-                      BLUIColors.primary[100]
-                    : // @TODO Currently neutral30 is #4242E4 and as per color pallete black[500] is #4142E4 Add Figma Variable
-                      // @ts-ignore
-                      BLUIColors.neutral[30]
-                : // @ts-ignore TODO
-                  BLUIColors.primary[100]; // default avatar is dark gray -> white text
+                    ? theme.colors.onPrimaryFilledContainer
```

Comment:
Hi @huayunh can you check if these color codes are correct? If not please provide me a Figma link of the default color. when we worked on this I didn't have a color code and we just chose close color code if you see removed line 342.